### ISSUE TITLE
Cannot load meshpy module

### ISSUE THREAD

**Issue by zeebok** [kromar/Blender_Addons#1]
Perhaps something changed I wasn't aware of with 2.7x versions of Blender but when I try to follow your instructions for installing, and try to activate the addon, I get an error:

```
 Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender\2.77\scripts\modules\addon_utils.py", line 349, in enable
    mod.register()
  File "C:\Program Files\Blender Foundation\Blender\2.77\scripts\addons\meshpy\__init__.py", line 64, in register
    bpy.utils.register_module(__name__)
  File "C:\Program Files\Blender Foundation\Blender\2.77\scripts\modules\bpy\utils\__init__.py", line 612, in register_module
    raise Exception("register_module(%r): defines no classes" % module)
Exception: register_module('meshpy'): defines no classes
```

**Comment by kromar** [comment]
hi,
have you put the module files located here: https://github.com/kromar/Blender_Addons/tree/master/addons_contrib/MeshPy/meshpy_windows 
in the "..\2.77\scripts\modules\meshpy" folder that needs to be in the modules? maybe this is causing your problem.
else, try to grab a updated version of the modules here: http://www.lfd.uci.edu/~gohlke/pythonlibs/ and extract them to the ..\2.77\scripts\modules\meshpy folder

**Comment by zeebok** [comment]
Double checked and grabbed updated modules, and no luck. Just to be sure, since this all seems to be case sensitive, I have copied the files in meshpy_windows into "..\2.77\scripts\modules\meshpy" (not capitalized to modules\MeshPy) 

Then the **init**.py, MeshPy.py, and MeshSlicer.py scripts are placed in "..\2.77\scripts\addons\meshpy"

This is correct? I tried having them all lowercase, which seems to cause Blender to error saying No Module named 'MeshPy' so I have been capitalizing the modules\MeshPy folder which led me to the error I started this thread over.
